src/main/telemetry/jetiexbus.c

```diff
@@ -173,74 +204,135 @@ void initJetiExBusTelemetry(void)
     jetiExTelemetryFrame[EXTEL_HEADER_LSN_LB] = 0x00;            // increment by telemetry count (%16) > only 15 values per device possible
     jetiExTelemetryFrame[EXTEL_HEADER_LSN_HB] = 0x00;
     jetiExTelemetryFrame[EXTEL_HEADER_RES] = 0x00;               // reserved, by default 0x00
+    
+    
+    // Check which sensors are available
+    
+    if (batteryConfig()->voltageMeterSource != VOLTAGE_METER_NONE) jetiExSensors[EX_VOLTAGE].parameter = EX_TYPE_SENSOR_ENABLED;
```

Comment:
Coding standards for Betaflight are to always use blocks for `if`, and to not use extra unneeded braces in conditions.

Reply:
Is corrected.